src/pages/home/report/ReportActionsView.js

```diff
@@ -275,7 +275,7 @@ class ReportActionsView extends React.Component {
     }
 
     fetchData() {
-        Report.fetchActions(this.props.reportID);
+        Report.readOldestAction(this.props.reportID);
```

Comment:
There are a few more changes that need to happen in this file, but this one is incorrect as we are not reading the oldest action here.

Reply:
I think I got it now, we need to call `ReadOldestAction` in the places we are calling `fetchActions` and passing an oldestAction

Reply:
Almost... we need to call `ReadOldestAction` when the user "reads the oldest action". Which is just another way of saying "pagination".

Reply:
yes, that makes sense.
I think we only have one place at the moment though